src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM_NUMBER = 20;
+    public static final double DEFAULT_RADIUS = 10.0;
```

Comment:
which access modifier should be considering that you use them only in this class?

Reply:
private